plugin-modernizer-core/src/main/resources/versions.properties

```diff
@@ -1 +1,5 @@
 openrewrite.maven.plugin.version = ${openrewrite.maven.plugin.version}
+jenkins.version = 2.440.2
```

Comment:
Why not 2.440.3 ? 

Perhaps I would be more specific on the property name ? Like jenkins.minimum.version

Reply:
You're right, it was a lame attempt at getting my `updatecli` script to make its first pull request afterward. 😊 
I'll correct it.